scipy/sparse/csgraph/_laplacian.py

```diff
@@ -20,6 +20,7 @@ def laplacian(
     form="array",
     dtype=None,
     symmetrized=False,
+    use_abs=False,
```

Comment:
Just a drive by comment: if we're adding a new argument, let's make it keyword only
```suggestion
    *,
    use_abs=False,
```